package.json

```diff
@@ -46,7 +46,7 @@
     "release": ". ./scripts/release.sh"
   },
   "peerDependencies": {
-    "swiper": "^5.2.0",
+    "swiper": "5.x || 6.x",
     "vue": "2.x"
   },
   "devDependencies": {
```

Comment:
Shouldn't the devDependency use swiper 6.x also?

Reply:
It is as you say. I'll fix it.